src/screens/ModelPage/ModelPage_Phase3.js

```diff
@@ -0,0 +1,241 @@
+        <MapMenuMobileWrapper>
+          <MapMenuItem
+            onClick={() => setMapOption(MAP_FILTERS.NATIONAL)}
+            selected={mapOption === MAP_FILTERS.NATIONAL}
+          >
+            United States
+          </MapMenuItem>
+          {stateId !== MAP_FILTERS.DC && (
+            <MapMenuItem
+              onClick={() => setMapOption(MAP_FILTERS.STATE)}
+              selected={mapOption === MAP_FILTERS.STATE}
+            >
+              {stateName}
+            </MapMenuItem>
+          )}
+        </MapMenuMobileWrapper>
+        <MapContentInner>
+          <MapWrapper visible={mapOption === MAP_FILTERS.NATIONAL}>
+            <Map
+              hideLegend={true}
+              setMapOption={setMapOption}
+              setMobileMenuOpen={setMobileMenuOpen}
+            />
+          </MapWrapper>
+
+          {stateId !== MAP_FILTERS.DC && (
+            <CountyMapAltWrapper visible={mapOption === MAP_FILTERS.STATE}>
+              <CountyMap
+                fill={
+                  projections.primary
+                    ? projections.getAlarmLevelColor()
+                    : '#e3e3e3'
+                }
+                stateSummary={stateSummary}
+                selectedCounty={selectedCounty}
+                setSelectedCounty={fullFips => {
+                  const county = _.find(
+                    US_STATE_DATASET.state_county_map_dataset[stateId]
+                      .county_dataset,
+                    ['full_fips_code', fullFips],
+                  );
+
+                  goTo(
+                    `/us/${stateId.toLowerCase()}/county/${
+                      county.county_url_name
+                    }`,
+                  );
+                  setMobileMenuOpen(false);
+                  setSelectedCounty(county);
+                }}
+              />
+            </CountyMapAltWrapper>
+          )}
+        </MapContentInner>
+      </MapContentWrapper>
+    );
+  };
+
+  return (
+    <Wrapper>
+      {metaTags}
+      <ContentWrapper>
+        {renderHeader()}
+        {projections && !projections.primary ? (
+          <div>
+            <StyledNoResultsWrapper>
+              <StyledNoResults>
+                <div>
+                  We either weren't able to get data for {selectedCounty.county}
+                  , {selectedCounty.state_code}, or no cases have yet been
+                  confirmed.
+                </div>
+                <div>
+                  We’re unable to produce our model for this county until we
+                  have data. Check back soon.
+                </div>
+                <div style={{ marginTop: '1rem' }}>
+                  View projections for{' '}
+                  <span onClick={() => goTo('/us/' + stateId.toLowerCase())}>
+                    {stateName}
+                  </span>
+                </div>
+              </StyledNoResults>
+            </StyledNoResultsWrapper>
+            {renderMapContent()}
+          </div>
+        ) : (
```

Comment:
Consider creating a component for the "no data" state (or create a render function). We can do that in a separate PR too.